src/Build/Resources/Strings.resx

```diff
@@ -1271,6 +1271,10 @@
     <value>MSB4067: The element &lt;{0}&gt; beneath element &lt;{1}&gt; is unrecognized.</value>
     <comment>{StrBegin="MSB4067: "}</comment>
   </data>
+  <data name="PropertyOutsidePropertyGroupInTarget" xml:space="preserve" Condition="$([MSBuild]::AreFeaturesEnabled('17.6'))">
+    <value>MSB4070: The property &lt;{0}&gt; beneath target &lt;{1}&gt; is unrecognized. Properties must be inside a &lt;PropertyGroup&gt; element.</value>
```

Comment:
rainersigwald pointed out that MSB4070 is already in use, so this should be merged into 4067 instead of being its own thing. (I think we pointed you towards making a new error; sorry for pointing you in two directions!)

Reply:
Sorry, I didn't search all anf found MSB4070 is used.  Yeah, we can merge the string to MSB4067<value>MSB4067: The element &lt;{0}&gt; beneath target &lt;{1}&gt; is unrecognized. If you intended this to be a property, it must be inside a &lt;PropertyGroup&gt; element.</value>

Reply:
I think we can change the error to MSB4073. Searched all and there are no duplicates

Reply:
If merged to original 4067, it seems not directllly to tell user this is a property

Reply:
It can still be a new, distinct string in this file with a distinct name, but use the same MSB4067 code as the other one. I would prefer to share the code, since if the user did _not_ intend to make a property, a search for the error code will continue to find any existing information.